docs/components/compact.md

```diff
@@ -39,9 +39,9 @@ Resolution - distance between data points on your graphs. E.g.
 - 5m - data point is every 5 minutes
 - 1h - data point is every 1h
 
-Keep in mind, that the initial goal of downsampling is not saving disk space (Read further for elaboration on storage space consumption). The goal of downsampling is providing an opportunity to get fast results for range queries of big time intervals like months or years. In other words, if you set `--retention.resolution-raw` less then `--retention.resolution-5m` and `--retention.resolution-1h` - you might run into a problem of not being able to "zoom in" to your historical data.
+Keep in mind, that the initial goal of downsampling is not saving disk space (Read further for elaboration on storage space consumption). The goal of downsampling is to provide an opportunity to get fast results for range queries of big time intervals like months or years. In other words, if you set `--retention.resolution-raw` less then `--retention.resolution-5m` and `--retention.resolution-1h` - you might run into a problem of not being able to "zoom in" to your historical data.
 
-To avoid confusion - you might want to think about `raw` data as about "zoom in" opportunity. Considering the values for mentioned options - always think "Will I need to zoom in to the day 1 year ago?" if the answer "yes" - you most likely want to keep raw data for as long as 1h and 5m resolution, otherwise you'll be able to see only downsampled representation of how your raw data looked like.
+To avoid confusion, you might want to think about `raw` data as about "zoom in" opportunity. When you are determining the values for - raw, - 5m, and - 1h,  consider if you might need to zoom in to this day 1 year ago, or similar. If you might, you most likely want to keep raw data for as long as the 1h and 5m resolution, otherwise you'll be able to see only a downsampled representation of what your raw data looked like.
```

Comment:
```suggestion
To avoid confusion, you might want to think about `raw` data as about "zoom in" opportunity. When you are determining the values for `-raw`, `-5m`, and `-1h`,  consider if you might need to zoom in to this day 1 year ago, or similar. If you might, you most likely want to keep raw data for as long as the 1h and 5m resolution, otherwise, you'll be able to see only a downsampled representation of what your raw data looked like.
```